webapp/src/main/java/com/box/l10n/mojito/entity/AssetExtraction.java

```diff
@@ -46,6 +47,7 @@
 public class AssetExtraction extends AuditableEntity {
 
   @JsonBackReference("asset")
+  @Schema(hidden = true)
```

Comment:
@JsonBackReference is not supported by Swagger, and these fields were included in the schema response

Reply:
it is not displayed in the HTTP response, but it shows up in the response schema